README.md

```diff
@@ -26,6 +26,8 @@ for a Python package.
   - Support to pre-commit
   - CI with github actions
   - Release workflow with semantic release and github actions
+  - You can choose the build system for your package, scicookie currently
```

Comment:
Maybe it would be better with something like this:

```
- Flexible build system selection: Choose between popular build systems like Poetry or Flit based on your preference.
```